src/main/java/com/gmail/nossr50/config/WorldBlacklist.java

```diff
@@ -3,91 +3,55 @@
 import com.gmail.nossr50.mcMMO;
 import org.bukkit.World;
 
-import java.io.*;
-import java.util.ArrayList;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
 
 /**
  * Blacklist certain features in certain worlds
  */
 public class WorldBlacklist {
-    private static ArrayList<String> blacklist;
+    private static final Set<String> blacklist = new HashSet<>();
     private final mcMMO plugin;
 
-    private final String blackListFileName = "world_blacklist.txt";
-
-    public WorldBlacklist(mcMMO plugin)
-    {
+    public WorldBlacklist(mcMMO plugin) {
```

Comment:
This parameter should probably be annotated as `@NotNull`
```suggestion
    public WorldBlacklist(@NotNull mcMMO plugin) {
```